torchinductor/triton_ops/autotune.py

```diff
@@ -147,13 +176,28 @@ def pointwise_heuristics(size_hints):
         return autotune(
             [
                 triton_config(size_hints, 64, 64),
```

Comment:
replace with 32 x 32?

Reply:
will fix in next pr, merging to unblock @pyjhzwh